scripts/solr_builder/psql-import-chunk.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+DUMP="$1"
+OFFSET="$2"
+COUNT="$3"
+
+time zcat "${DUMP}" | \
+tail -n +$((OFFSET+1)) | \
+head -n $COUNT | \
+sed --expression='s/\\u0000//g' | \
```

Comment:
⛔

Reply:
I *think* with a `rebase -i` and some well placed `pick`s, this file will drop out of the PR entirely.